src/recordlinker/schemas/pii.py

```diff
@@ -441,27 +439,24 @@ def feature_iter(self, feature: Feature) -> typing.Iterator[str]:
                     yield str(race)
         elif attribute == FeatureAttribute.TELECOM:
             for telecom in self.telecom:
-                if telecom.system is None:
-                    yield telecom.value.strip().lower()
-                    continue
-
-                handlers = Telecom.get_system_handlers()
-
-                if telecom.system in handlers:
-                    value = getattr(telecom, handlers[telecom.system])()
-                    if value:
-                        yield value
-
+                if telecom.system == "phone":
+                    # Use national number for comparison
+                    phone = normalize_text(str(phonenumbers.parse(telecom.value).national_number))
```

Comment:
I understand why we are calling `normalize_text` here, but why are we reparsing the value with phonenumbers?  Shouldn't we only have to do that once during in the `validate_and_normalize_telecom` method?

Reply:
Ah thanks for that explanation, that's helpful!